### DISCUSSION TITLE
Vehicle composition skin permeation model

### DISCUSSION THREAD

**Discussion by MelanieMartha26** [Open-Systems-Pharmacology/Forum#1074] [Questions & Problems]
Dear everyone,

I am using the skin permeation model derived from the study of Dancik et al. in MoBi.

I was wondering how you would approach implementing a vehicle that is a mixture of both water and ethanol. Would you just adjust the vehicle solubility and keep the vehicle type at 1? And what would you do regarding its volatility. Normally I would keep it at 0 for water and at 1 for ethanol.

Thank you very much in advance, and if anything is unclear, I do not mind explaining my question further.

Best regards,

Mèlanie

**Comment by abdullahhamadeh** [comment]
Hi @MelanieMartha26 

Yes, that is how I would normally model a volatile vehicle using Dancik's model - i.e. you assume that some of the vehicle remains on the surface due to the water component not evaporating.  Of course, there are contexts where the water may also evaporate relatively early in the course of the experiment, so I think it is important to also explore the case where volatile vehicle option = 1.

Please also [see this publication by our group](https://www.mdpi.com/1999-4923/13/6/807), where we explore the assumption (absent in Dancik's 2013 model) that the permeant ceases to flow into the SC (i.e. it remains on the skin surface) once the vehicle has evaporated completely.
